templates/forms/22-contact.html.twig

```diff
@@ -91,7 +91,7 @@
 
                     <!-- form -->
                     {# For Twig global variables see config/packages/<environment>/twig.yaml #}
-                    <form class="govuk-!-width-three-quarters" action="{{ web_to_case_post_url }}" method="POST">
+                    <form class="govuk-!-width-three-quarters" action="/contact/sendToSalesforce" method="GET">
```

Comment:
I just noticed the form method of GET here. That doesn't seem right?

Reply:
Contact form changes has been tested on dev and regards about the "GET", I will change it to a POST and see if there will be any impact.